src/Synth/OscilGen.cpp

```diff
@@ -1715,15 +1716,15 @@ FILTER(lp2)
 
 FILTER(hp2)
 {
-    if(par == 1)
+    if(par >= 1)
```

Comment:
Unless there's a good reason, 1.0 == 1.0, so there's no problem with a direct comparison of values.
As far as I can see the source of the 'par' value is going to produce exactly 1.0 in the case this is handling.

Reply:
OK, but the compiler (which cannot know the background) complained that comparing floats with == was unsafe. We can either use a trick or just [disable `-Wfloat-equal`](https://stackoverflow.com/a/11422298), what do you think?

Reply:
Resolved by disabling `-Wfloat-equal`